src/RazorSdk/SourceGenerators/RazorSourceGenerator.cs

```diff
@@ -19,13 +19,16 @@ public partial class RazorSourceGenerator : IIncrementalGenerator
 
         public void Initialize(IncrementalGeneratorInitializationContext context)
         {
-            // determine if we should suppress this run and set all the inputs to cached if so
-            var isGeneratorSuppressed = context.AnalyzerConfigOptionsProvider.Select(GetSuppressionStatus);
+            var analyzerConfigOptions = context.AnalyzerConfigOptionsProvider;
+            var parseOptions = context.ParseOptionsProvider;
+            var compilation = context.CompilationProvider;
```

Comment:
Do we do compute any caches for compilations that will run even if we don't have any other input files? I see the tag helpers from references and such. Doing that up front isn't necessarily bad, but just wanted to confirm it's running when you think it's running.

Reply:
Everything else has a 'skip if there are no razor files' so it means we'll bail straight away for everything else too.